src/pages/jj/Main/Aside/Story/StoryList/StoryList.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+// APIKEY
+import api from '../../../../config/api';
+
+class StoryList extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      story: [],
```

Comment:
```suggestion
    this.state = {
      stories: [],
```
배열의 경우 복수형 명사를 사용해주시는게 좋습니다